src/components/slices/hero-banner.js

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
+import { graphql } from 'gatsby'
+import { GatsbyImage, StaticImage } from 'gatsby-plugin-image'
+import { PrismicRichText, PrismicLink } from '@prismicio/react'
+import {
+  BiArrowBack
+} from 'react-icons/bi'
+
+import * as sty from './hero-banner.module.scss'
+
+export const HeroBanner = ({ slice }) => {
+
+  return (
+    <section className={sty.heroBanner} style={{ paddingBottom: 0 }}>
+      <div className={`${sty.heroContainer} ${location.pathname === '/' ? sty.homeHero : ''}`}>
+      <div className={sty.logoWrap}>
+        <StaticImage
+          src="../../images/designstudio_logo.png"
+          width={378}
+          height={182}
+          layout="constrained"
+          className={sty.image}
```

Comment:
import error from this since it isn't used in the scss file
![image](https://github.com/heynovaio/designstudio/assets/159569575/06a019c4-ad65-486f-aece-638e1654dfd9)